### ISSUE TITLE
main script for attacks on esc50

### ISSUE THREAD

**Issue by ChrisNick92** [magcil/audio-adversarial-attacks#25]
Create a source script in `finetune` and a `.yaml` for setting the hyperparameters for fine-tuning and attacking a chosen model.

**For finetuning:** The .yaml should take as input

1. The number of training folds (e.g. 1, 2, 3, 4).
2. The test fold (e.g 5).
3. Number of epochs, patience, learning rate etc.
4. Path to save the pt file.

Output: Report Classification accuracy

**For attacking:** 

1. Algorithm (PSO or DE)
2. hyperparameters
3. etc

Use logs.
